### ISSUE TITLE
a_end_mcr_configuration on partner VXC causes continuous recreation of resource

### ISSUE THREAD

**Issue by hougaardj** [megaport/terraform-provider-megaport#29]
The a_end_mcr_configuration defined for the general VXC has force new for everything, which then destroys my VXC every time i make some other change.

CSP VXC https://github.com/megaport/terraform-provider-megaport/blob/main/schema_megaport/megaport_azure_connection.go#L42
Geneal VXC mcr configuration https://github.com/megaport/terraform-provider-megaport/blob/main/schema_megaport/megaport_vxc.go#L114

Workaround:
```
resource "megaport_azure_connection" "azure_cnx" {
  ...

  lifecycle {
    ignore_changes = [a_end_mcr_configuration]
  }
}
```

**Comment by hougaardj** [comment]
Most of the config in this field also seems to have ForceNew to create the resource, which would imply the connection gets cut and then recreated. E.g. manipulation of MED in/out and BFD would be nice to update in-place.

**Comment by mega-alex** [comment]
This has been addressed in the `v1.0.0` provider.
